src/egg-to-dios-dsl.rkt

```diff
@@ -54,18 +33,45 @@
 ; Functional expression -> (name, list of DSL instructions)
 (define (egg-to-dios e)
   (match e
-    [(egg-get name idx) e]
+    [0
+     (define zero (new-name 'z))
```

Comment:
I think this needs to be ``Z` to be consistent with the prelude, we can add it to some shared location